sda/cmd/api/api_test.go

```diff
@@ -374,6 +360,42 @@ func (suite *TestSuite) SetupTest() {
 	Conf.API.DB, err = database.NewSDAdb(Conf.Database)
 	assert.NoError(suite.T(), err)
 
+	Conf.Broker = broker.MQConf{
+		Host:     "localhost",
+		Port:     mqPort,
+		User:     "guest",
+		Password: "guest",
+		Exchange: "sda",
+		Vhost:    "/sda",
+	}
+	Conf.API.MQ, err = broker.NewMQ(Conf.Broker)
+	assert.NoError(suite.T(), err)
+
+}
+
+func (suite *TestSuite) SetupTest() {
+	log.Print("Setup DB for my test")
```

Comment:
Do we really want this `Print`?

Reply:
That block already existed in another place, but yes the print should not be there.